programs/vault/src/strategy/mango.rs

```diff
@@ -0,0 +1,10 @@
+use anchor_lang::prelude::*;
+use std::str::FromStr;
+
+pub fn get_mango_program_id() -> Pubkey {
```

Comment:
Since port finance, solend get program id function is in `base.rs`, shall we group them all in `base.rs` and remove this file ?

Reply:
Fixed